proxy/http/remap/PluginFactory.cc

```diff
@@ -259,6 +272,15 @@ PluginFactory::findByEffectivePath(const fs::path &path, bool dynamicReloadEnabl
   return PluginDso::loadedPlugins()->findByEffectivePath(path, dynamicReloadEnabled);
 }
 
+bool
+PluginFactory::isGlobalPluginDSOEnabled(const fs::path &path) const
```

Comment:
`isGlobalPluginDSOEnabled` feels nonsensical here, we don't actually enable or disable global plugins. This actually shows if the DSO that is used as a remap plugin will be reloadable. May be `isPluginDSOReloadEnabled`?